README.md

```diff
@@ -1,2 +1,39 @@
 # what-is-an-arg-paper
-Manuscript and code for the "What is an ARG?" paper
+Manuscript and code for the "A general and efficient representation of ARGS" paper
+
+Create the paper by running `make` in the top level.
+
+## Creating graphics
+
+Install the necessary python libraries:
+
+```
+python -m pip install -r requirements.txt
+```
+
+Then to recreate plots used in the paper, type `make clean && make`
+
+### Re-running inferences
+
+To also re-run the software used for the inference plots, the
+necessary inference tools need to be installed by running the makefile
+in the tools directory (note that this
+requires `python2` to be installed, to run ARGweaver conversion software)
```

Comment:
Also to run ARGweaver itself, right?

Reply:
I think ARGweaver is a C++ binary, so arg-sample (the main routine) can be run without any python

Reply:
Yeah - the inference command is

```
tools/argweaver/bin/arg-sample --sites examples/Kreitman_SNP.sites --output examples/argweaver_output/arg-sample --overwrite --smc-prime --popsize 1e6 --mutrate 5.49e-09 --recombrate 8.4e-09 --randseed 111 --iters 3 --sample-step 10000 --no-compress-output
```

Which doesn't require python2

Reply:
Ah, I thought argweaver itself was py2. Cool, all good.